### DISCUSSION TITLE
Add option to print Captures

### DISCUSSION THREAD

**Discussion by elrac** [Orange-OpenSource/hurl#1975] [Ideas]
Currently it is possible to print captures by turning on verbose mode, however that prints a lot of other information. How difficult would it be to have a flag, --print-captures, that just prints the captures just like they are printed in verbose mode?

**Comment by jcamiel** [comment]
Hi @elrac 

Can you solve your use case with [`--json`](https://hurl.dev/docs/manual.html#json)? JSON option outputs a structured view of a Hurl run, with requests, responses, captures etc...

For instance:

```
$ hurl --json foo.hurl | jq
{
  "entries": [
    {
      "request": {
        "method": "GET",
        "url": "http://localhost:8000/captures"
      },
      "response": {
        "status": 200,
        "captures": [
          {
            "name": "param1",
            "query": {
              "type": "header",
              "name": "header1"
            }
          },
          {
            "name": "param2",
            "query": {
              "type": "header",
              "name": "header2"
            },
            "filters": [
              {
                "type": "regex",
                "expr": "Hello (.*)!"
              }
            ]
          },

...

      }
    }
  ]
}
```

**Comment by elrac** [comment]
My specific case is that I have a set of endpoints and when new ones are added I have to make sure they work and also see how large the output is. I'm doing this by using 

[Captures]
count: jsonpath "$" count

on the last line. I don't really care about the rest of the output other than that one line and the results from -test. Currently if I use --verbose it gives me the that output near the bottom of the output, but I don't care about the rest. I had seen where other people had requested different outputs, like a print statement or something and was thinking that just printing the captures like in the verbose mode would be an easy way to do it.
